tap-snapshots/test-arborist-build-ideal-tree.js-TAP.test.js

```diff
@@ -12474,6 +12476,394 @@ ArboristNode {
 }
 `
 
+exports[`test/arborist/build-ideal-tree.js TAP do not ERESOLVE on peerOptionals that are ignored anyway case a > must match snapshot 1`] = `
+&ref_1 Node {
+  "children": Map {
+    "@isaacs/testing-peer-optional-conflict-a-x" => Node {
+      "children": Map {},
+      "dev": false,
+      "devOptional": false,
+      "dummy": false,
+      "edgesIn": Set {
+        Edge {},
+      },
+      "edgesOut": Map {
+        "@isaacs/testing-peer-optional-conflict-a-z" => Edge {},
+      },
+      "errors": Array [],
+      "extraneous": false,
+      "fsChildren": Set {},
+      "hasShrinkwrap": false,
+      "integrity": "sha512-GYIjOCQ5BFWl0caylWvjYOLrs0JZq0SkAY0WhNegXPXehob14/DjVD3LxwUWIKzgbhfr345ig46H0jFEmdV66Q==",
+      "inventory": Inventory {},
+      "legacyPeerDeps": false,
+      "linksIn": Set {},
+      "location": "node_modules/@isaacs/testing-peer-optional-conflict-a-x",
+      "name": "@isaacs/testing-peer-optional-conflict-a-x",
+      "optional": false,
+      "path": "{CWD}/test/fixtures/peer-optional-eresolve/a/node_modules/@isaacs/testing-peer-optional-conflict-a-x",
+      "peer": false,
+      "realpath": "{CWD}/test/fixtures/peer-optional-eresolve/a/node_modules/@isaacs/testing-peer-optional-conflict-a-x",
```

Comment:
heads up, looks like `cleanSnapshot` is failing to clean these up in all windows ci targets

Reply:
Yeah, I missed something there.  Gotta get that fixed before landing, might be a real issue.